packages/p2p-v2/src/pages/my-profile/screens/MyProfileStats/MyProfileStatsLimitNotification.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import MyProfileStatsLimitModal from './MyProfileStatsLimitModal';
+import './MyProfileStatsLimitNotification.scss';
+
+type TMyProfileStatsLimitNotificationProps = {
+    buyLimit: number;
+    currency: string;
+    sellLimit: number;
+};
+
+const MyProfileStatsLimitNotification = ({ buyLimit, currency, sellLimit }: TMyProfileStatsLimitNotificationProps) => {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+
+    return (
+        <>
+            <div className='p2p-v2-my-profile-stats-limit-notification'>
+                <Text color='less-prominent' lineHeight='sm' size='xs'>
+                    Want to increase your daily limits to{' '}
+                    <span className='p2p-v2-my-profile-stats-limit-notification--bold'>
+                        {buyLimit} {currency}{' '}
+                    </span>{' '}
+                    (buy) and{' '}
+                    <span className='p2p-v2-my-profile-stats-limit-notification--bold'>
+                        {sellLimit} {currency}{' '}
+                    </span>{' '}
```

Comment:
Why not use the `Text` component here @nada-deriv, since you already  installed `@deriv-com/ui`? 🤔 
```suggestion
                    <Text className='p2p-v2-my-profile-stats-limit-notification--bold'>
                        {buyLimit} {currency}{' '}
                    </Text>{' '}
                    (buy) and{' '}
                    <Text className='p2p-v2-my-profile-stats-limit-notification--bold'>
                        {sellLimit} {currency}{' '}
                    </Text>{' '}
```

Reply:
yeah.. will udpate it.. before running prettier it actually looked cleaner with span 😅